hack/codespell.sh

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+
```

Comment:
curious why `hack/codespell.sh` seems to have a `.sh` extension but `hack/golangci-lint` doesn't